cli/command/stack/remove.go

```diff
@@ -13,56 +14,63 @@ import (
 )
 
 type removeOptions struct {
-	namespace string
+	namespaces []string
 }
 
-func newRemoveCommand(dockerCli *command.DockerCli) *cobra.Command {
+func newRemoveCommand(dockerCli command.Cli) *cobra.Command {
 	var opts removeOptions
 
 	cmd := &cobra.Command{
 		Use:     "rm STACK",
 		Aliases: []string{"remove", "down"},
 		Short:   "Remove the stack",
```

Comment:
Thank you. I will change that line. 
Perhaps I need to create a PR on docker/docker.github.io as well, after the changes to the CLI docs are approved?